pom.xml

```diff
@@ -648,7 +648,8 @@
                 <module>spring-boot-jasypt</module>
                 <module>spring-boot-keycloak</module>
                 <module>spring-boot-logging-log4j2</module>
-                <module>spring-boot-mvc</module>
+				<module>spring-boot-mvc</module>
```

Comment:
Can we convert the indents back to spaces here and below before merging?

Reply:
Hi Kevin, all done. I'm sorry I did it in 2 commits instead of one